File: my-workout-api/src/muscle/muscle.controller.ts
import { Controller, Get } from '@nestjs/common';
import { MuscleService } from './muscle.service';

@Controller('muscle')
export class MuscleController {

    constructor(
        private readonly muscleService: MuscleService
    ) {
    }

    @Get()
    findAllMuscle() {
        return this.muscleService.findAllMuscle();
    }

}

File: my-workout-api/src/main/java/com/avernet/myworkoutapi/muscle/MuscleController.java
package com.avernet.myworkoutapi.muscle;

import jakarta.annotation.Resource;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

@RestController
@RequestMapping("muscle")
public class MuscleController {

    @Resource
    MuscleService muscleService;

    @GetMapping
    List<MuscleDropdown> findAllMuscle() {
        return muscleService.findAllMuscle();
    }
}
